src/components/QuestDetailsWrapper/index.tsx

```diff
@@ -152,8 +158,8 @@ export function QuestDetailsWrapper(props: QuestDetailsWrapperProps) {
       'Connect Steam account'
     ),
     questType: {
-      REPUTATION: t('quest.type.reputation', 'Reputation'),
-      PLAYSTREAK: t('quest.type.playstreak', 'Play Streak')
+      PLAYSTREAK: t('quest.type.playstreak', 'Play Streak'),
+      GAME: gameName
```

Comment:
game isn't a quest type. did you mean to add this prop somewhere else?

Reply:
These are two different badges, one is the type of quest, and the other is the game name.